app/main.py

```diff
@@ -2,0 +1,22 @@
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file = parts[1]
+    destination_file = parts[2]
+
+    if source_file == destination_file:
+        return
```

Comment:
Code Style: It would be better to raise an exception or print an error message if the source file is the same as the destination file, instead of silently returning.